astarte-setup.sh

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env bash
+
+# (C) Copyright 2024, SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -eEuo pipefail
+
+# Run the setup script non interactively.
+ASTARTE_SETUP_NON_INTERACTIVE=${ASTARTE_SETUP_NON_INTERACTIVE:-false}
+# Directory where to setup (clone) the Astarte repository.
+ASTARTE_SETUP_CLONE_DIR=${ASTARTE_SETUP_CLONE_DIR:-"$HOME"}
```

Comment:
```suggestion
ASTARTE_SETUP_CLONE_DIR=${ASTARTE_SETUP_CLONE_DIR:-"$PWD"}
```

Shouldn't we prefer the current working directory?

Reply:
The home directory would make it less likely for some user to create multiple Astarte instances all in slightly different folders.